### PR TITLE
Description

### PR DESCRIPTION
**Pull request by Pluriscient**
Solves #6

### CODE REVIEW

**Comment by stefsiekman** [comment]
Okay three points:

- Please provide a more meaningful title
- Please provide a description
- Describe how you tested your code

(JK LOL)

File: predicate.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ class Predicate {
     }
 
     markdownArgumentTable() {
-        if (!this.args)
+        if (!this.args || this.args.every((val) => !val.description))
```
Comment (stefsiekman) [inline predicate.js:229]: Nice compact solution!

**Review by stefsiekman — CHANGES_REQUESTED** [review]
Please give a report on testing first, the implementation itself looks great.

**Comment by Pluriscient** [comment]
Testing has been done

File: predicate.js (hunk 1 above)
Comment (Pluriscient) [reply predicate.js:229]: Thanks!

**Comment by stefsiekman** [comment]
> Testing has been done

@Pluriscient 

Thanks!

**Review by stefsiekman — APPROVED** [review]

**Pull request merged**